de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/AccessSqlParser.java

```diff
@@ -1,77 +1,84 @@
+ * See the GNU General Public License for more details. *
+ * You should have received a copy of the GNU General Public License along *
+ * with this program; if not, write to the Free Software Foundation, Inc., *
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA. *
+ * For the text or an alternative of this public license, you may reach us *
+ * ComPiere, Inc., 2620 Augustine Dr. #245, Santa Clara, CA 95054, USA *
+ * or via info@compiere.org or http://www.compiere.org/license.html *
  *****************************************************************************/
 package org.compiere.model;
 
 import java.util.ArrayList;
 import java.util.StringTokenizer;
+
 import org.slf4j.Logger;
+
 import de.metas.logging.LogManager;
 
 /**
- *	Parse FROM in SQL WHERE clause
- *	
- *  @author Jorg Janke
- *  @version $Id: AccessSqlParser.java,v 1.3 2006/07/30 00:58:36 jjanke Exp $
+ * Parse FROM in SQL WHERE clause
+ * 
+ * @author Jorg Janke
+ * @version $Id: AccessSqlParser.java,v 1.3 2006/07/30 00:58:36 jjanke Exp $
  * 
  * @author Teo Sarca, SC ARHIPAC SERVICE SRL
- * 			<li>BF [ 1652623 ] AccessSqlParser.getTableInfo(String) - tablename parsing bug
- * 			<li>BF [ 1964496 ] AccessSqlParser is not parsing well JOIN CLAUSE
- * 			<li>BF [ 2840157 ] AccessSqlParser is not parsing well ON keyword
- * 				https://sourceforge.net/tracker/?func=detail&aid=2840157&group_id=176962&atid=879332
+ *         <li>BF [ 1652623 ] AccessSqlParser.getTableInfo(String) - tablename parsing bug
+ *         <li>BF [ 1964496 ] AccessSqlParser is not parsing well JOIN CLAUSE
+ *         <li>BF [ 2840157 ] AccessSqlParser is not parsing well ON keyword
+ *         https://sourceforge.net/tracker/?func=detail&aid=2840157&group_id=176962&atid=879332
  */
 public class AccessSqlParser
 {
 	/**
-	 *	Base Constructor.
-	 *	You need to set the SQL and start the parsing manually.
+	 * Base Constructor.
+	 * You need to set the SQL and start the parsing manually.
 	 */
-	public AccessSqlParser ()
+	public AccessSqlParser()
 	{
-	}	//	AccessSqlParser
+	}	// AccessSqlParser
 
 	/**
-	 *	Full Constructor 
-	 *	@param sql sql command
+	 * Full Constructor
+	 * 
+	 * @param sql sql command
 	 */
-	public AccessSqlParser (String sql)
+	public AccessSqlParser(String sql)
 	{
 		setSql(sql);
-	}	//	AccessSqlParser
-
-	/**	FROM String			*/
-	private static final String		FROM = " FROM ";
-	private static final int		FROM_LENGTH = FROM.length();
-	private static final String		WHERE = " WHERE ";
-	private static final String		ON = " ON ";
-
-	/**	Logger				*/
-	private Logger	log = LogManager.getLogger(getClass());
-	/**	Original SQL			*/
-	private String		m_sqlOriginal;
-	/**	SQL Selects			*/
-	private String[]	m_sql;
-	/**	List of Arrays		*/
-	private ArrayList<TableInfo[]>	m_tableInfo = new ArrayList<TableInfo[]>(); 
+	}	// AccessSqlParser
+
+	private static final String FROM = " FROM ";
+	private static final String from = " from ";
```

Comment:
instead of "from" i would suggest something like FROM_LOWERCASE or FROM_LC... same for "where" and "on".

Reply:
done